packages/ui-kit/src/design-system.js

```diff
@@ -129,9 +130,14 @@ export const dimensions = {
   widths: {
     pageContent: pageWidth,
     pageContentSmall: pageWidthSmall,
-    pageContentWithMargings: `calc(${pageWidth} + 32px * 2)`,
-    pageContentSmallWithMargings: `calc(${pageWidthSmall} + 16px * 2)`,
-    pageContentWithMargingsAndPageNavigation: `calc(${pageWidth} + 32px * 2 + ${navbarWidth})`,
+    pageContentWide: `calc(${pageWidthSmall} + ${asideColumnWidth})`,
+    pageContentWideMax: `calc(${pageWidth} + ${asideColumnWidth} + 32px)`,
+    pageContentWithMargins: `calc(${pageWidth} + 32px * 2)`,
+    pageContentSmallWithMargins: `calc(${pageWidthSmall} + 16px * 2)`,
+    pageContentWideWithMargins: `calc(${pageWidthSmall} + ${asideColumnWidth} + 32px * 3)`,
+    pageContentWideWithMarginsMax: `calc(${pageWidth} + ${asideColumnWidth} + 32px * 3)`,
+    pageContentWithMarginsAndPageNavigation: `calc(${pageWidth} + 32px * 2 + ${navbarWidth})`,
+    pageContentWideWithMarginsAndPageNavigation: `calc(${pageWidthSmall} + ${asideColumnWidth} + 32px * 3 + ${navbarWidth})`,
```

Comment:
Are you using all those new sizes?

Reply:
Good question :-).  They aren't all new, just the ones with "Wide" in them.  Not sure why all lines are marked as new in the diff. I'll cross check, would not be a surprise if some turned out to not be used.

Reply:
@emmenko  I found out why the all look new:  I refactored a spelling error "Margings->Margins"  in the existing names, so they are all changed.  Plus, two were actually not used any more, I removed them.